Subsetting.Rmd

```diff
@@ -10,14 +10,14 @@ rownames(mtcars) <- NULL
 
 R's subsetting operators are fast and powerful. Mastering them allows you to succinctly perform complex operations in a way that few other languages can match. Subsetting in R is easy to learn but hard to master because you need to internalise a number of interrelated concepts:
 
-* There are six types of data you can subset.
+* There are six ways to subset atomic vectors.
```

Comment:
Here I'm trying to imply that there are six different types of thing that `y` can be in `x[y]`. But I'm using type in a different way to the rest of the book, and this is just the intro, so I think your wording is fine.